publish-service/src/test/java/com/ericsson/eiffel/remrem/publish/service/MessageServiceRMQImplUnitTest.java

```diff
@@ -170,8 +172,24 @@ public void testCreateExchangeIfNotExistingDisable() throws RemRemPublishExcepti
         assertEquals(Expected, jarray.toString());       
     }
 
+    @Test public void testPublisherConfirmsTimeoutException() throws Exception {
+        rmqHelper.getRabbitMqPropertiesMap().get(protocol).setWaitForConfirmsTimeOut(1L);
+        rmqHelper.getRabbitMqPropertiesMap().get(protocol).init();
+        JsonArray jarray = new JsonArray();
+        String body = FileUtils.readFileToString(new File("src/test/resources/EiffelActivityFinishedEvent.json"));
+        MsgService msgService = PublishUtils.getMessageService(protocol, msgServices);
+        while (!(jarray.toString().contains("Time out waiting for ACK"))) {
```

Comment:
Will it finally happen? What if there's a really fast RabbitMQ? Then the test may hang...